### DISCUSSION TITLE
VS Code Themes

### DISCUSSION THREAD

**Discussion by D-Upchurch** [nss-day-cohort-47/C47-Discussions#17] [Look what I learned!]
Thanks to Salma and Vishal, I found that you can change your theme on VS Code. Some of these themes are incredibly helpful to show differences in your code between variables, functions, methods, etc. Might help someone (like me) understand what each part in javascript actually is. I'm currently using monokai which also makes certain code display in italics (parameters, const, let) as well as alters colors. 

You can easily access this using ' ctrl+k then ctrl+t ' on windows, I'm sure the mac version is similar.

**Comment by srcrank** [comment]
You can also find additional themes to download in the extensions tab on VS code! Two of my favorite downloadable theme packs are Mayukai and Noctis, both have some really awesome color combinations!!

**Reply by vishus12** [reply]
Highly recommend the Mayukai theme!

**Comment by Chadwell-Clark** [comment]
### Though not VSCode themes I have found a few extensions that I like:
**CSS Peek** allows you to look at your css while working on your html. Will show you a peek of a class, id or tags css
**Indent Rainbow** gives you an easy way to see your indentations and how code is lining up 
**Prettier** Makes your code prettier and uniform
**REST client** Though I have not used it yet it is basically POSTMAN in VSCode GET, POST etc to your API
*Bryan* mentioned **Better Comments** and **Bracket Pair Colorizer2** which are both very helpful -so thanks **Bryan!**
As far as VSCode themes I do like **Monokai** ... so I will have to check out **Mayukai**

**Comment by SidneyCrandall** [comment]
https://vscodethemes.com/

If you would like to see what some of the themes look like in different tabs of VS Code, (html, js, css)

**Comment by EugeneGohh** [comment]
For me, I always use One Dark Pro, it's much easier to see and easier on your eye. 

Feel free to check this out -> [One Dark Pro Extension](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)

If you want file icon example you would like JS icon on your file, do check this out:

[VS Code File Icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
